### DISCUSSION TITLE
NO_SYNTH_CELL_LIST

### DISCUSSION THREAD

**Discussion by young03600** [The-OpenROAD-Project/OpenLane#2179] [Q&A] [answered]
Hi,
According to the title of question.
I'd like to know what's the reason that we need to specify the no synth cells like the cells, 
e.g., sky130_fd_sc_hd__o2bb2a_1, sky130_fd_sc_hd__a22o_1, sky130_fd_sc_hd__and2_1...., listed in the deafult no_synth.cells.

Thanks in advance.

**Comment by maliberty** [comment, marked as answer]
The _1 sized cells are the smallest and can sometimes give difficulty to the router due to their small pin shapes.  You can enable them and see if they work ok or not (the router does improve over time).
